src/ostorlab/runtimes/local/app/oxo.py

```diff
@@ -1,14 +1,199 @@
-from typing import Optional
+from typing import Optional, List
```

Comment:
update please

Reply:
the `|` annotation wont work here since we have python 3.9 in the workflow.